inst/shiny-server/server/ontology.R

```diff
@@ -0,0 +1,188 @@
+
+# Populate dropdowns for algorithms (in case of bbob) or suites (in case of NG),
+# Only enabled when no study is selected
+observe({
+  req(input$Ontology.Source)
+
+  if (input$Ontology.Study != 'None') {
+    return()
+  }
+
+  input_source <- input$Ontology.Source
+  if (input_source == "Nevergrad") {
+    available_suite <- get_ontology_var("Suite", input_source)
+
+    if (is.null(available_suite) || length(available_suite) == 0) {# TODO: the alert msg should be updated
+      shinyjs::alert("No connection could be made with the OPTION ontology. Please try again later.")
+      shinyjs::disable('Ontology.Load')
+      return()
+    }
+
+    updateSelectInput(session, 'Ontology.NG_Suite', choices = available_suite, selected = available_suite[[1]])
+    shinyjs::disable('Ontology.Load')
+    return()
+  }
+
+  # available_functions <- get_ontology_var("Fid", input_source)
+  # available_dimensions <- get_ontology_var("DIM", input_source)
+  available_algorithms <- get_ontology_var("AlgId", input_source)
+  # available_iid <- get_ontology_var("Iid", input_source)
+
+  if (is.null(available_algorithms) || length(available_algorithms) == 0) {# TODO: the alert msg should be updated
+    shinyjs::alert("No connection could be made with the OPTION ontology. Please try again later.")
+    shinyjs::disable('Ontology.Load')
+    return()
+  }
+  # updateSelectInput(session, 'Ontology.Functions', choices = available_functions, selected = available_functions[[1]])
```

Comment:
Perhaps, we could delete those commented lines if we do not need those in the future?